compiler-rt/cmake/Modules/AddCompilerRT.cmake

```diff
@@ -680,15 +680,16 @@ macro(add_custom_libcxx name prefix)
 
   ExternalProject_Add(${name}
     DEPENDS ${name}-clobber ${LIBCXX_DEPS}
-    PREFIX ${CMAKE_CURRENT_BINARY_DIR}/${name}
+    PREFIX ${prefix}
     SOURCE_DIR ${LLVM_MAIN_SRC_DIR}/../runtimes
-    BINARY_DIR ${prefix}
+    BINARY_DIR ${prefix}/build
```

Comment:
Was this an expected change? If so, doesn't that mean the `-isystem` path in fuzzer's CMakeLists.txt should include the `/build/` prefix?

Reply:
No, the install directory is ${prefix}, so it uses the installed headers now instead of the build tree.

Reply:
So adding the `/build` suffix is intended then? We're seeing issues where certain C++ headers aren't found and it looks like someone else has commented the same as well.

Reply:
Yes this is intentional - and the dependency for RTFuzzer is still there so I am not sure why it is failing for you:

```
    target_compile_options(RTfuzzer.${arch} PRIVATE -isystem ${LIBCXX_${arch}_PREFIX}/include/c++/v1)
    add_dependencies(RTfuzzer.${arch} libcxx_fuzzer_${arch}-install)
```

Reply:
Yes I saw that. In our case, the headers were defined under `${LIBCXX_${arch}_PREFIX}/build/include/c++/v1`, hence my suspicion on `/build` whether that was intentional But perhaps, it might a configuration issue on our end. For now, I've disabled building fuzzer since it's not needed for our internal workloads.